client/src/utils.js

```diff
@@ -74,7 +74,7 @@ function getAffectedFilesText( pageIds ) {
 
 	fileIds = pageIds.filter( ( id ) => {
 		return id.startsWith( 'File:' );
-	} );
+	} ).map( fileId => fileId.replace( 'File:', '' ) );
```

Comment:
nit-pick... 
I would do it in the next loop:
```js
text = text + `* {{lf|${id.replace( 'File:', '' )}}}\n`;
```
no need to change what you have though as the functionality is the same (and I would argue that perhaps your version is cleaner anyways).